lldb/test/API/linux/aarch64/gcs/main.c

```diff
@@ -50,5 +63,7 @@ int main() {
   }
 
   // By now we should have one memory region where the GCS is stored.
-  return 0; // Set break point at this line.
+  gcs_signal(); // Set break point at this line.
```

Comment:
The previously added `test_gcs_region` will stop here and then kill the process instead of continuing. So it will work as it did before.